packages/frontend/amp/components/elements/VideoFacebookBlockComponent.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+export const VideoFacebookBlockComponent: React.FC<{
```

Comment:
Minor, but I changed the naming of these in a separate PR to remove the redundant 'BlockComponent' suffix - see https://github.com/guardian/dotcom-rendering/pull/515. `FacebookVideo` is probably okay in this case. 

(And ditto for the other elements you've created.)

Reply:
👍